users/tests/test_views.py

```diff
@@ -132,3 +134,33 @@ def test_user_send_activate_email_view(self):
         client.force_authenticate(user=self.test_user)
         response = client.post(reversed_url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+
+    def test_update_skills(self):
+        url = 'users-api:update_skills'
+        reversed_url = reverse(url)
+        client = APIClient()
+        client.force_authenticate(user=self.test_user)
```

Comment:
The test class is using `APITestCase`, so the client in `self.client` is already an instance of `APIClient`. So you can use just `self.client` instead of creating the new `client` variable. I know that the other tests are done that way but maybe they could be updated.

```suggestion
        self.client.force_authenticate(user=self.test_user)
```

Reply:
Awesome, I didn't know that, I can refactor these do just do self.client instead if you think that's cleaner.

Reply:
Yeah sure, I think that it would be better but this way is not that bad. Whatever the maintainers prefer.

Maybe a new issue could be added to refactor the other tests later.

Reply:
`self.client` refers to test case class's client where just `client` will refer to the APIclient instance ! they might be different think i guess !

Reply:
Yeah, that would be the case if the test was using django's default `TestCase`, but the test class `class AccountTests(APITestCase):` is using `APITestCase` from drf, that uses `APIclient` for the client.

Reply:
alright got it! thanks